index.js

```diff
@@ -54,6 +65,21 @@ module.exports = function (src, dest, opts) {
 		return new Promise(function pipeToDest(resolve, reject) {
 			var write = fs.createWriteStream(dest, {flags: opts.overwrite ? 'w' : 'wx'});
 
+			read.on('data', function () {
+				if (size === 0) {
+					return;
+				}
+				var written = write.bytesWritten;
+				var percent = written / size;
+				progressEmitter.emit('progress', {
+					src: src,
+					dest: dest,
```

Comment:
These just output what the user supplied. That's rarely a good idea. The user could have supplied an absolute path or a relative path. We should ensure it's consistent. Absolute paths would probably be best. The user can then just `path.relative()` if they want relative paths. (This normalization should obviously be done at the start of the function and not in each progress event).

@YurySolovyov @schnittstabil Thoughts?